### DISCUSSION TITLE
Integrate Wavesurfer-js with shopify?

### DISCUSSION THREAD

**Discussion by mikeypoo313** [katspaugh/wavesurfer.js#3224] [Q&A]
I was wondering if someone could explain how to get the player to work on shopify. I want to have different audio players assigned to different product pages. My reference is [[cymatics.fm](cymatics.fm)](https://cymatics.fm/pages/phoenix-beta-free-download). The player works in vscode, but I can't get it to work on my shopify store.

**Comment by fahadbinhasanrafi** [comment]
I can help you regarding that. Paid task :)

**Comment by pk27602** [comment]
> I can help you regarding that. Paid task :)

How much are you charging?

**Reply by fahadbinhasanrafi** [reply]
If you need the Landing Page exactly same as Cymatics (you sent), I can design it with Gempages/Pagefly. It will cost you $500.
If the Landing Page is already created, I will just integrate the Wavesurfer only with $320.

**Reply by samplethisdotnet** [reply]
hey bro did you ever find this out? I'm sure if you have a gem pages element that has the code importnat it's really easy to duplicae I just can't figure out how to get the code to work on gempages in the first place...

**Comment by fahadbinhasanrafi** [comment]
Hey There. Hope you are doing great today. Yes. It will cost around 180 for
4 wavesurfer waveform.

**Reply by samplethisdotnet** [reply]
It's just a element on gempages custom code and them you use the audio file link within shopify content. I'm willing to pay for the knowledge of how to do so but $180 for adding that element is outrageous it's copy and paste.

**Reply by pk27602** [reply]
> It's just a element on gempages custom code and them you use the audio file link within shopify content. I'm willing to pay for the knowledge of how to do so but $180 for adding that element is outrageous it's copy and paste.

Hello mate,, i found it. contact me and i will send the code for free
https://darkwinter-music.com/
